test/boot_storage_test.rb

```diff
@@ -92,4 +92,34 @@
       expect(result).to include("/dev/vda")
     end
   end
+
+  describe ".multipath_mapping" do
+    before do
+      mock_disk_partition
+      # force reinit every time
+      allow(subject).to receive(:checkCallingDiskInfo).and_return(true)
+      # mock getting mount points as it need whole libstorage initialization
+      allow(Yast::Storage).to receive(:GetMountPoints).and_return("/" => "/dev/vda1")
+      # mock for same reason getting udev mapping
+      allow(::Bootloader::UdevMapping).to receive(:to_mountby_device) do |arg|
+        arg
+      end
+    end
+
+    it "returns empty map if there is no multipath" do
+      target_map_stub("storage_lvm.rb")
```

Comment:
yeah, "own" is wrong term, I mean it is real data to prevent regressions, so unless target map format is changed, then it should not be changed.